src/RA_httpthread.cpp

```diff
@@ -245,7 +245,7 @@ BOOL RAWeb::DoBlockingRequest(RequestType nType, const PostArgs& PostData, Docum
     {
         if (response.size() > 0)
         {
-            JSONResponseOut.Parse(DataStreamAsString(response));
+			JSONResponseOut.Parse( DataStreamAsString( response ).c_str());
```

Comment:
these are tabs. you can tell by highlighting in the browser - you can't select the individual spaces.

Reply:
Thanks for pointing it out.